client/src/Components/MyApplications/index.jsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Container } from 'react-bootstrap';
+import ApplicationCard from '../CommonComponents/OfferCard';
+import applications from '../utils/myApplications';
+import './style.css';
 
 export default function MyApplications() {
   return (
     <>
-      <h1>my applications</h1>
-      <UnderConstruction />
+      <Container className="page__container">
+        <div className="my-applications__title">
+          <span>My Applications</span>
+        </div>
+        {applications
+          ? applications.map(item => {
+              if (item.status !== 'active') {
```

Comment:
why did you filter out the `active` status? :thinking:

Reply:
ohh, I'll remove this condition
thanks